xbmc/messaging/ApplicationMessenger.h

```diff
@@ -32,20 +32,22 @@
 #define TMSG_PLAYLISTPLAYER_ADD           TMSG_MASK_PLAYLISTPLAYER + 3
 #define TMSG_PLAYLISTPLAYER_CLEAR         TMSG_MASK_PLAYLISTPLAYER + 4
 #define TMSG_PLAYLISTPLAYER_SHUFFLE       TMSG_MASK_PLAYLISTPLAYER + 5
-#define TMSG_PLAYLISTPLAYER_GET_ITEMS     TMSG_MASK_PLAYLISTPLAYER + 6
-#define TMSG_PLAYLISTPLAYER_PLAY_SONG_ID  TMSG_MASK_PLAYLISTPLAYER + 7
-#define TMSG_PLAYLISTPLAYER_INSERT        TMSG_MASK_PLAYLISTPLAYER + 8
-#define TMSG_PLAYLISTPLAYER_REMOVE        TMSG_MASK_PLAYLISTPLAYER + 9
-#define TMSG_PLAYLISTPLAYER_SWAP          TMSG_MASK_PLAYLISTPLAYER + 10
-#define TMSG_PLAYLISTPLAYER_REPEAT        TMSG_MASK_PLAYLISTPLAYER + 11
-#define TMSG_MEDIA_PLAY                   TMSG_MASK_PLAYLISTPLAYER + 12
-#define TMSG_MEDIA_STOP                   TMSG_MASK_PLAYLISTPLAYER + 13
+#define TMSG_PLAYLISTPLAYER_IS_SHUFFLED   TMSG_MASK_PLAYLISTPLAYER + 6
```

Comment:
Why not put this after `TMSG_PLAYLISTPLAYER_GET_REPEAT` to minimize the impact on this list?

Reply:
I'd put new stuff below `TMSG_MEDIA_STOP` and not change any of the existing values.

Reply:
I had added `TMSG_PLAYLISTPLAYER_IS_SHUFFLED` next to `TMSG_PLAYLISTPLAYER_SHUFFLE` (same for `TMSG_PLAYLISTPLAYER_GET_REPEAT` and `TMSG_PLAYLISTPLAYER_REPEAT`). I pushed a new version where they are at the end.